File: app/src/main/java/com/song/deviceinfo/ui/others/OthersViewModel.java
package com.song.deviceinfo.ui.others;

import android.content.Context;

import com.song.deviceinfo.info.OthersInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/9/22.
 */
public class OthersViewModel extends NormalViewModel {

    public List<Pair<String, String>> getOthersInfo(Context context) {
        return OthersInfo.getOthersInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/others/OthersViewModel.kt
package com.song.deviceinfo.ui.others

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.OthersInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/9/22.
 */
class OthersViewModel : NormalViewModel() {
    fun getOthersInfo(context: Context?): List<Pair<String, String>> {
        return OthersInfo.getOthersInfo(context!!)
    }
}
